app/src/main/java/com/xmartlabs/scasas/criminalintent/ui/crime/list/CrimeAdapter.java

```diff
@@ -67,6 +67,7 @@ public int getItemCount() {
     public CrimeHolder(View itemView, OnTappedListener<Crime> onCrimeTappedListener) {
       super(itemView);
       ButterKnife.bind(this, itemView);
+      solvedCheckBox.setEnabled(false);
```

Comment:
Also, you can set a check box as "read only" by disabling the *clickable* property: http://stackoverflow.com/a/11213759/1165181 But this is just fine